Client/Frontend/Browser/Tab Management/TabManager.swift

```diff
@@ -764,6 +765,8 @@ class TabManager: NSObject, FeatureFlaggable, TabManagerProtocol {
             // The default tab configurations also need to change.
             self.configuration.preferences.javaScriptCanOpenWindowsAutomatically = allowPopups
             self.privateConfiguration.preferences.javaScriptCanOpenWindowsAutomatically = allowPopups
+            
+            self.configuration.mediaTypesRequiringUserActionForPlayback = AutoplayAccessors.getMediaTypesRequiringUserActionForPlayback(self.profile.prefs);
```

Comment:
Could you remove the `;` at the end?
Also I think this should be applied on the `privateConfiguration` config too? So we could apply this the same as `allowPopups` is applied.